### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by bryankj5** [iklein99/homebridge-smartthings#201] [Q&A]
Hello,
I had been using the tonesto homebridge smartapp on smartthings but sometime in the past week it finally went away, I knew it was going to deprecated but it was still working for me. Actually it's still working but the app is gone so I can't add new devices. I'm running homebridge and have a Vivint and Harmony plugin running, but I can't seem to get my devices to show up in homekit. I can add the bridge but no devices show up. I've created a token and added it to homebridge, but still no dice. Any help getting devices to show up would be much appreciated!

**Comment by tarostegui** [comment]
Just want to second that I'm having this exact same issue.  I can see all my ST devices in the HomeBridge application but none of them are flowing through to HomeKit.  

I as also formerly running the tones to HomeBridge smartapp and am attempting to switch over.

**Reply by coryabooth** [reply]
Have you tried clearing the cache in HomeBidge settings?

**Comment by iklein99** [comment]
If you can see the devices in Homekit, then the plugin is working.  Try removing and adding Homebridge to the home app.  Also, try restarting the device you are using as your home hub.

If you want to send me your detailed logs, I'll have a look to see if there are any errors loading devices.  Follow the instructions below.
[Instructions for Producing a Detailed Log.pdf](https://github.com/iklein99/homebridge-smartthings/files/12388630/Instructions.for.Producing.a.Detailed.Log.pdf)
